### DISCUSSION TITLE
Hey!

### DISCUSSION THREAD

**Discussion by Ino5559** [sonic3XE/evades-region-editor#14] [Ideas]
You can make cybots for cyber castre but i wanted kill cybot but i found Normal.and if you can make hero cybot .pls make bosses in cyber castle  and add hero cybot is gonna very cool

**Comment by sonic3XE** [comment]
most cc2 minibosses and bosses are implemented and may sometimes crash due to unimplemented entity that are attempting to shoot or spawn
